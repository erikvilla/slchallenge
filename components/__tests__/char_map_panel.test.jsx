import React from 'react';
import renderer from 'react-test-renderer';

import CharMapPanel from '../char_map_panel';
import mockData from '../__mockData__/char_map_panel';

describe('Enzyme', () => {
  it('Renders a table for 100 persons', () => {
    const charMap = mockData;
    const component = renderer.create(<CharMapPanel charMap={charMap} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
