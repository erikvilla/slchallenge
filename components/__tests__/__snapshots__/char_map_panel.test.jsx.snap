// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Enzyme Renders a table for 100 persons 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
  >
    <div
      aria-controls="panel1a-content"
      aria-disabled={false}
      aria-expanded={false}
      className="MuiButtonBase-root MuiExpansionPanelSummary-root"
      id="panel1a-header"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <div
        className="MuiExpansionPanelSummary-content"
      >
        <p
          className="MuiTypography-root makeStyles-heading-2 MuiTypography-body1"
        >
          Show email char count
        </p>
      </div>
      <div
        aria-disabled={false}
        aria-hidden={true}
        className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
      </div>
    </div>
    <div
      className="MuiCollapse-container MuiCollapse-hidden"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper"
      >
        <div
          className="MuiCollapse-wrapperInner"
        >
          <div
            aria-labelledby="panel1a-header"
            id="panel1a-content"
            role="region"
          >
            <div
              className="MuiExpansionPanelDetails-root"
            >
              <table
                className="MuiTable-root"
              >
                <thead
                  className="MuiTableHead-root"
                >
                  <tr
                    className="MuiTableRow-root MuiTableRow-head"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-head"
                      scope="col"
                    >
                      Letter
                    </th>
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
                      scope="col"
                    >
                      Count
                    </th>
                  </tr>
                </thead>
                <tbody
                  className="MuiTableBody-root"
                >
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      e
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      112
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      a
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      92
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      n
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      85
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      i
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      81
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      o
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      71
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      r
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      65
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      l
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      64
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      s
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      45
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      .
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      37
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      t
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      36
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      c
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      34
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      _
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      30
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      h
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      29
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      y
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      25
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      d
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      22
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      u
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      22
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      m
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      20
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      M
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      18
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      1
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      17
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      3
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      17
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      C
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      15
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      2
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      13
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      9
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      13
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      7
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      13
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      S
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      13
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      4
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      13
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      5
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      12
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      A
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      12
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      6
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      12
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      J
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      12
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      E
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      12
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      R
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      11
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      k
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      11
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      K
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      11
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      b
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      10
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      L
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      10
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      p
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      9
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      G
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      9
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      H
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      9
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      f
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      8
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      v
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      8
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      8
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      8
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      B
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      8
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      w
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      7
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      0
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      7
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      g
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      7
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      D
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      6
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      O
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      5
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      P
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      4
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      z
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      3
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      T
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      3
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      V
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      2
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      W
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      2
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      X
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      2
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      N
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      2
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      j
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      1
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      U
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      1
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      x
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      1
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      Y
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      1
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      F
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      1
                    </td>
                  </tr>
                  <tr
                    className="MuiTableRow-root"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body"
                      scope="row"
                    >
                      Z
                    </th>
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                    >
                      1
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
